docs/components/optimize/userguide/processes.md

```diff
@@ -7,9 +7,9 @@ description: "The processes section gives you an overview of all the processes a
 ## Overview
 
 The processes section gives you an overview of all the processes and their status on a single page.
-This section also allows you to set a process owner to take responsibility of a process and time goals to track process performance.
+This section also allows you to set a process owner to take responsibility of a process as well as creating time and quality kpis to track process performance. Additionally, a **Management Dashboard** can be found, which integrates data from all processes in one view.
```

Comment:
```suggestion
This section also allows you to set a process owner to take responsibility of a process as well as viewing time and quality KPIs to track process performance. Additionally, a **Management Dashboard** can be found, which integrates data from all processes in one view.
```
small suggestion but it's not possible to create these from the process page is it?